app/lib/common/utils/utils.dart

```diff
@@ -272,6 +272,8 @@ enum NetworkStatus { NotDetermined, On, Off }
 // ignore: constant_identifier_names
 enum RoomVisibility { Public, Private, SpaceVisible }
 
+enum AttachmentType { camera, image, audio, video, file }
```

Comment:
don't we also have a `location` type :thinking: ? well, future support I guess..

Reply:
Yes indeed, but only included now what's supported! ;)

Reply:
then, arguably, there should be an "other" option because later code might submit that and if we see a "other" we want to display a "please upgrade to the latest version to view and interact with this"-kinda-message.